tree/30_generic_methods/monitoring_parameter.cf

```diff
@@ -0,0 +1,50 @@
+#####################################################################################
+# Copyright 2013 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# @name Monitoring parameter
+# @description Add a monitoring parameter to a node
```

Comment:
We could use a `@documentation` section here, with a usage example. It should explain what a monitoring parameter is and it is used for. See other methods for syntax (the content itself is in matkdown)

Reply:
add a (requires a monitoring plugin)